tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -184,59 +102,104 @@ $ mkdir -p ./proto/gogoproto
+
+```txt
+Telescope 1.0.3
+? [pkg] which packages do you want to support? (Press <space> to select, <a> to toggle all, <i> to invert selection)
+❯◯ amino
+ ◯ akash
+ ◯ bcna
+ ...
+ ◯ cosmos
+ ...
+```
+
+Say `cosmos`'s selected and entered, there'll be cosmos folder and its dependencies(like gogoproto and google, etc) in `myLib/proto` folder.
+</CodeGroupItem>
+<CodeGroupItem title="NonInteractive">
+
+You can add wanted proto packages after `telescope install`, then you don't have to select them one by one.
 
 ```sh
-$ ls ./proto/myChain | xargs -I {} protoc \
-  --plugin="./node_modules/ts-proto/protoc-gen-ts_proto" \
-  --ts_proto_out="./client/src/types/generated" \
-  --proto_path="./proto" \
-  --ts_proto_opt="esModuleInterop=true,forceLong=long,useOptionals=messages" \
-  myChain/{}
+$ npx telescope install @protobufs/cosmos@0.0.10  @protobufs/confio
 ```
 
+Then there'll be the packages and its dependencies(like gogoproto and google, etc) in myLib/proto folder.
+
 </CodeGroupItem>
+</CodeGroup>
+
+### Compilation
 
-<CodeGroupItem title="Docker">
+You can now compile the Protobuf files using telescope.
 
 ```sh
-$ ls ./proto/myChain | xargs -I {} \
-    docker run --rm -i \
-    -v $(pwd):/project -w /project \
-    ts-protoc \
-    --plugin="/usr/local/lib/node_modules/ts-proto/protoc-gen-ts_proto" \
-    --ts_proto_out="./client/src/types/generated" \
-    --proto_path="./proto" \
-    --ts_proto_opt="esModuleInterop=true,forceLong=long,useOptionals=messages" \
-    myChain/{}
+$ npx telescope transpile
 ```
 
-Where `/usr/local/lib/node_modules` is the result of the query:
+You should now see some `.ts` files generated in `./src/codegen`. These are the real source files used in your application.
+
+By default, the command takes `./proto` as input folder and `./src/codegen` as output folder. You can config in and out folders and the way telescope generating code by using options like the examples:
 
 ```sh
-$ docker run --rm -it \
-    --entrypoint npm \
-    ts-protoc \
-    root --global
+# Telescope takes chain1 folder as input,
+# and generate files in 'gen/src' folder.
+$ npx telescope transpile --protoDirs ./proto --outPath gen/src
 ```
 
-This shows where `ts-proto` was installed globally.
+Each time `telescope transpile` has been ran, a config file `.telescope.json` will be created in the folder where it's running:
+
+```json
+//.telescope.json
+{
+  "protoDirs": [
+    "../../proto/chain2"
+  ],
+  "outPath": "gen/src",
+  "options": {
+    // telescope options
+    "addTypeUrlToDecoders": true,
+    "addTypeUrlToObjects": true,
+
+    "typingsFormat": {
+        "num64": "bigint",
+        "customTypes": {
+            "useCosmosSDKDec": true
+        }
+    },
+    ...
+  }
+}
+```
 
-</CodeGroupItem>
+You can modify the config file according to [telescope options](https://github.com/cosmology-tech/telescope#options), and then use the file by the option `--config`:
```

Comment:
This link could be added in `1-cosmjs-objects` too.

Reply:
Good catch! Done!